.tests/config/config.yaml

```
@@ -128,4 +128,22 @@ voc:
     - N501Y
     - E484K
     - K417N
-    - H69_V70del
\ No newline at end of file
+    - H69_V70del
+  
+# assemblers used for assembly comparison
+assemblers_for_comparison:
+  - megahit-std
+  - megahit-meta-large
+  - megahit-meta-sensitive
+  - trinity
+  - velvet
+  - metaspades
+  - coronaspades
+  - spades
+  - rnaviralspades
+
+# assemblers used for sequencing method 
+assembler_shotgun:
+  megahit-std
+assembler_amplicon:
+  metaspades
```

also change all instances
```suggestion
assemblers_used:
 - shotgun: megahit-std
 - amplicon: metaspades
```